ush/tropcy_relocate_extrkr.sh

```diff
@@ -489,19 +489,19 @@ mv ${vdir}/tempvit.nonameless ${vdir}/vitals.${symd}${dishh}
 # tracking program.
 #--------------------------------------------------------------#
 
-ymdh6ago=$( ${NDATE:?} -6 ${CENT}${symd}${dishh})
-syy6=$(echo ${ymdh6ago} | cut -c3-4)
-smm6=$(echo ${ymdh6ago} | cut -c5-6)
-sdd6=$(echo ${ymdh6ago} | cut -c7-8)
-shh6=$(echo ${ymdh6ago} | cut -c9-10)
-symd6=${syy6}${smm6}${sdd6}
-
-ymdh6ahead=$( ${NDATE:?} 6 ${CENT}${symd}${dishh})
-syyp6=$(echo ${ymdh6ahead} | cut -c3-4)
-smmp6=$(echo ${ymdh6ahead} | cut -c5-6)
-sddp6=$(echo ${ymdh6ahead} | cut -c7-8)
-shhp6=$(echo ${ymdh6ahead} | cut -c9-10)
-symdp6=${syyp6}${smmp6}${sddp6}
+ymdh6ago=$(date --utc +%Y%m%d%H -d "${CENT}${symd}${dishh} - 6 hours")
```

Comment:
Not too sure, but definitely needs a space between `YYYYMMDD` and `HH`
```suggestion
ymdh6ago=$(date --utc +%Y%m%d%H -d "${CENT}${symd} ${dishh} - 6 hours")
```